pkg/cloud/api/check.go

```diff
@@ -171,3 +171,41 @@ func checkSchema(t reflect.Type) error {
 
 	return nil
 }
+
+// canConvertToT checks if we can convert type From to type To. Conversion is
```

Comment:
I find this confusing -- what is the purpose of the path `p` here? It is not mentioned at all in the docstring.

Reply:
This is only for better error reporting. This way we have nested fields info and without the path we can only log field name and type but if this is in nested structs we won't have this information

Reply:
Added comment about path parameter. Does it make sense?